### PR TITLE
:sparkles:(llm/lld): add dynamic learn more for LS activation

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - LS activation LLD and LLM

### 📝 Description

Add a field to the feature flag so we can dynamically change the learn more url when activate ledger sync.  
The learmore link will appears only if a value is provided
The UI has changed on both LLD and LLM

Good and bad case integration tests added

The new field is `learnMoreLink` 

LLD : 
https://github.com/user-attachments/assets/92fae350-6a76-4b0f-920c-efbfa6aeb3e9

LLM : 
https://github.com/user-attachments/assets/bc0fec2d-8b05-4673-aa08-33feb6ec6eaf

### ❓ Context

- **JIRA or GitHub link**: [LIVE-14081](https://ledgerhq.atlassian.net/browse/LIVE-14081) & [LIVE-14082](https://ledgerhq.atlassian.net/browse/LIVE-14082)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14081]: https://ledgerhq.atlassian.net/browse/LIVE-14081?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mcayuelas-ledger** [comment]
this UI looks weird no ? https://www.figma.com/design/uI7PKnUK8nAZ0USyiEF3wQ/Ledger-Sync?node-id=10584-23812&node-type=frame&t=VcdchRYDsTGycXRq-0

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Activation/Actions.tsx [hunk 1]
```diff
@@ -10,9 +10,16 @@ import {
 type Props = {
   onPressSyncAccounts: () => void;
   onPressHasAlreadyCreatedAKey: () => void;
+  onPressLearnMore: () => void;
+  hasLearnMoreLink?: boolean;
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Activation/Actions.tsx:14]: You can simplify just by `onPressLearnMore` as optional ?

**Review by mcayuelas-ledger — CHANGES_REQUESTED** [review]

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Activation/index.tsx [hunk 2]
```diff
@@ -5,19 +5,25 @@ import { Flex, Text } from "@ledgerhq/native-ui";
...
+  const walletSyncFF = useFeature("llmWalletSync");
+  const learnMoreLink = walletSyncFF?.params?.learnMoreLink;
 
   useInitMemberCredentials();
 
   const onPressSyncAccounts = () => onSyncMethodPress();
 
   const onPressHasAlreadyCreatedAKey = () => onSyncMethodPress();
 
+  const onPressLearnMore = () => learnMoreLink && Linking.openURL(learnMoreLink);
```
Comment (themooneer) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Activation/index.tsx:25]: you may have an eslint problem here for `no-unused-expressions` rule. For a better readability and code maintainability the better is to use an if bloc and invoke your openURL function there.

File: apps/ledger-live-desktop/src/newArch/features/WalletSync/screens/Activation/01-CreateOrSynchronizeStep.tsx [hunk 3]
```diff
@@ -15,6 +17,11 @@ type Props = {
 export default function CreateOrSynchronizeStep({ goToCreateBackup, goToSync }: Props) {
   const { colors } = useTheme();
   const { t } = useTranslation();
+  const ledgerSyncFF = useFeature("lldWalletSync");
+  const learnMoreUrl = ledgerSyncFF?.params?.learnMoreLink;
+  const hasLearnMoreLink = !!learnMoreUrl;
+
+  const onLearnMore = () => hasLearnMoreLink && openURL(learnMoreUrl);
```
Comment (themooneer) [inline apps/ledger-live-desktop/src/newArch/features/WalletSync/screens/Activation/01-CreateOrSynchronizeStep.tsx:24]: same here https://github.com/LedgerHQ/ledger-live/pull/7825/files#r1762619896

**Comment by LucasWerey** [comment]
> this UI looks weird no ? https://www.figma.com/design/uI7PKnUK8nAZ0USyiEF3wQ/Ledger-Sync?node-id=10584-23812&node-type=

Changed for

Screenshot 2024-09-17 at 10 56 11

**Review by themooneer — APPROVED** [review]

**Review by mcayuelas-ledger — APPROVED** [review]

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**